### ISSUE TITLE
Ubuntu 22.04.01 Minimal Install - Error: Could not install package 'python'

### ISSUE THREAD

**Issue by war59312** [hakwerk/labca#57]
Looks like it fails on Ubuntu 22.04.1 minimal install.

Getting: `Error: Could not install package 'python'`

Full log:

```
will@labca:~$ curl -sSL https://raw.githubusercontent.com/hakwerk/labca/master/install | bash

  [✗] Not running as root
  [✓] Run using sudo

  [✓] Running as root
  [✓] Package 'git' is installed
  [✓] Package 'sudo' is installed
  [✓] Created user 'labca'
  [✓] Clone https://github.com/hakwerk/labca/ to /home/labca/labca
  [.] Restarting updated version of install script...

  [✓] Running as root
  [✓] Package 'git' is installed
  [✓] Package 'sudo' is installed
  [✓] User 'labca' already exists
  [✓] Update git repository in /home/labca/labca
FQDN (Fully Qualified Domain Name) for this PKI host (users will use this in their browsers and clients)? [labca] labca.mydomain.com
  [✓] Determine web address
  [✓] Setup admin application
  [✓] Configure the admin application
  [✓] Software is up-to-date
  [✓] Package 'apt-transport-https' is installed
  [✓] Package 'ca-certificates' is installed
  [✓] Package 'curl' is installed
  [✓] Package 'gnupg2' is installed
  [✓] Package 'net-tools' is installed
  [✓] Package 'software-properties-common' is installed
  [✓] Package 'tzdata' is installed
  [✓] Package 'ucspi-tcp' is installed
  [✓] Package 'zip' is installed
  [.] Install package 'python'...
  Error: Could not install package 'python'

will@labca:~$ sudo apt install python
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package python is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  python2-minimal:i386 python2:i386 python2-minimal python2 dh-python 2to3 python-is-python3

E: Package 'python' has no installation candidate
will@labca:~$ sudo apt install python3
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
python3 is already the newest version (3.10.4-0ubuntu2).
python3 set to manually installed.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
```

Looks like should be looking for python3 instead of python.

**Comment by hakwerk** [comment]
Should be fixed now in the latest release (v22.09.1)

**Comment by war59312** [comment]
Confirmed, fixed.

Thank you!

```
will@labca:~$ curl -sSL https://raw.githubusercontent.com/hakwerk/labca/master/install | bash

  [✗] Not running as root
  [✓] Run using sudo
[sudo] password for will:

  [✓] Running as root
  [✓] Package 'git' is installed
  [✓] Package 'sudo' is installed
  [✓] User 'labca' already exists
  [✓] Update git repository in /home/labca/labca
  [✓] Determine web address
  [✓] Setup admin application
  [✓] Configure the admin application
  [✓] Software is up-to-date
  [✓] Package 'apt-transport-https' is installed
  [✓] Package 'ca-certificates' is installed
  [✓] Package 'curl' is installed
  [✓] Package 'gnupg2' is installed
  [✓] Package 'net-tools' is installed
  [✓] Package 'software-properties-common' is installed
  [✓] Package 'tzdata' is installed
  [✓] Package 'ucspi-tcp' is installed
  [✓] Package 'zip' is installed
  [✓] Package 'python3' is installed
  [✓] Package 'docker-ce' is installed
  [✓] Binary 'docker-compose' is installed
  [✓] Static web pages
  [✓] Create self-signed certificate
  [✓] Clone https://github.com/letsencrypt/boulder/ to /home/labca/gopath/src/github.com/letsencrypt/boulder
  [✓] Boulder checkout 'release-2022-09-14'
  [✓] Setup boulder configuration folder
  [✓] Configure the boulder application
  [✓] Cleaning up obsolete files
  [✓] Restart docker containers and service
 done!


========

Congratulations! LabCA is now installed and should be available at https://labca.erynandwill.com
Please go there now to finish the setup. Note that a TEMPORARY (7 days) self-signed certificate
is used; as part of the setup verification a new certificate will be issued.
```

And the LabCA web page loading fine.
